lib/Data/ContentTypeData.php

```diff
@@ -33,6 +33,13 @@ class ContentTypeData extends ContentTypeUpdateStruct implements NewnessCheckabl
      */
     protected $fieldDefinitionsData = [];
 
+    /**
+     * Language Code of currently edited contentTypeDraft.
+     *
+     * @var string|null
+     */
+    public $usedLanguageCode = null;
```

Comment:
Why not just `languageCode`?

Reply:
No particular reasons, thought that would give more emphasis to it. Going to change it.